extensions/amp-list/0.1/amp-list.js

```diff
@@ -920,8 +920,9 @@ export class AmpList extends AMP.BaseElement {
   loadMoreCallback_(opt_reload = false) {
     if (!!this.loadMoreSrc_) {
       this.element.setAttribute('src', this.loadMoreSrc_);
+      this.loadMoreSrc_ = null;
```

Comment:
This line is the bugfix. Basically clear the loadMoreSrc and not try to refetch if a fetch is already in progress.

Reply:
Could you add that as a comment?

Reply:
Added.